types/base/error.d.ts

```diff
@@ -0,0 +1,15 @@
+export declare class YoniusError {
+    constructor(message: string, code?: number)
+    isClient() : boolean
+    isServer() : boolean
```

Comment:
@BeeMargarida there's seems to be an extra space here. Are you the one adding it or prettier?
```suggestion
    isClient(): boolean
    isServer(): boolean
```